pkg/transformer/join.go

```diff
@@ -0,0 +1,18 @@
+package transformer
+
+import "github.com/pingcap/parser/ast"
+
+type (
+	JoinTransformer struct {
```

Comment:
JoinTransformer is ambiguous, how about CombinedTransformer?

Reply:
Seems that JoinTransformer is used to connect two Transformer?

Reply:
Resolved